product/alerts/rss/vmware_vms.yml

```diff
@@ -9,7 +9,7 @@ item_title: |-
     <script>
         if rec.host != nil
           "#{rec.name} on Host #{rec.host.name}"
-        elsif ! rec.repositories.empty?
+        elsif rec.respond_to?(:repositories) && ! rec.repositories.empty?
```

Comment:
@bmclaughlin repositories have been removed from the codebase, can you double check that we might not even need this elsif block at all.